packages/table/src/tableBody.tsx

```diff
@@ -85,6 +85,7 @@ export class TableBody extends React.Component<ITableBodyProps, {}> {
                 onFocus={this.props.onFocus}
                 onSelection={this.props.onSelection}
                 onSelectionEnd={this.handleSelectionEnd}
+                requireMetaKeyToDeselect={true}
```

Comment:
non-configurable? why even have a prop?

Reply:
Because I added it at the beginning of this work. And then realized by the end of it that I didn't need it to be configurable. Will remove.